### DISCUSSION TITLE
is this is the correct usage of next.js app and pages directories?

### DISCUSSION THREAD

**Discussion by SandroAKHV** [vercel/next.js#56984] [App Router] [answered]
I'm working on next.js v13 project which uses /pages directory instead of /app, I just added /app directory to use its new features for dynamic sitemap rendering and only for that, routing still happens in /pages,  is this acceptable practice ?

**Comment by icyJoseph** [comment, marked as answer]
Yeah, you can incrementally move into App router from Pages, so having them both is to be expected.

Doing SPA transitions between App and Pages Routes, does trigger a hard navigation though.
